Runtime/Tracking/Follow/Modifier/Property/Rotation/TransformPositionDifferenceRotation.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Zinnia.Tracking.Follow.Modifier.Property.Rotation
+{
+    using UnityEngine;
+    using Malimbe.XmlDocumentationAttribute;
+    using Malimbe.PropertyValidationMethod;
+    using Malimbe.PropertySerializationAttribute;
+    using Zinnia.Extension;
+    using Zinnia.Data.Type;
+
+    /// <summary>
+    /// Updates the transform rotation of the target to match the difference in position of the source.
+    /// </summary>
+    public class TransformPositionDifferenceRotation : PropertyModifier
+    {
+        /// <summary>
+        /// The drag applied to the rotation.
+        /// </summary>
+        [Serialized, Validated]
+        [field: DocumentedByXml]
+        public float AngularDrag { get; set; } = 1f;
+
+        /// <summary>
+        /// Determines which axes to rotate.
+        /// </summary>
+        [Serialized, Validated]
+        [field: DocumentedByXml]
+        public Vector3State FollowOnAxis { get; set; } = Vector3State.True;
+
+        /// <summary>
+        /// The current angular velocity the rotation is applying to the target.
+        /// </summary>
+        public Vector3 AngularVelocity { get; protected set; }
+        /// <summary>
```

Comment:
```suggestion

        /// <summary>
```
I think we always "group" members by accessibility using newlines.